lib/mqtt.ts

```diff
@@ -17,6 +17,7 @@ import crt_native = require('./binding');
 
 import * as io from "./io";
 import { TextEncoder } from 'util';
+import ResourceSafety = require('./resource_safety')
```

Comment:
You can do `import { ResourceSafe } from './resource_safety';` for more consistent style, then you don't have to qualify the access later in the file.

Reply:
import { using } from .....